docs: update references to the ECS variant for GA (#1637)
bottlerocket-os/bottlerocket: CHANGELOG.md

With this release, the aws-ecs-1 variant has graduated from preview status and is now generally available.
It's been updated to include Docker 20.10.

> Additionally, a new [Bottlerocket ECS Updater](https://github.com/bottlerocket-os/bottlerocket-ecs-updater/) is now available to help provide automated updates to Bottlerocket.

Reviewer: ```suggestion
The new [Bottlerocket ECS Updater](https://github.com/bottlerocket-os/bottlerocket-ecs-updater/) is available to help provide automated updates.
```

Author: Done
